src/OpenLoco/Core/LocoFixedVector.hpp

```diff
@@ -79,4 +79,77 @@ namespace OpenLoco
             return Iter(endAddress, endAddress);
         }
     };
+
+    template<typename ValueType, size_t Count>
+    class FixedVector
```

Comment:
This adds a bit of boilerplate. Can we express the other FixedVector in terms of this one, or the other way around?

Reply:
By the end of this pr the first one will not be in use. So I'll remove it and rename this one to take the old name. (At least that's the plan..)